Riot/Modules/Room/RoomViewController.m

```diff
@@ -5497,8 +5558,13 @@ - (void)reactionsMenuViewModel:(ReactionsMenuViewModel *)viewModel didRemoveReac
     }];
 }
 
+
+BOOL debugEnableNewRoomRendering = TRUE;
```

Comment:
This boolean switches which rendering style is used for chats -- whether it's the default renderer (which doesn't support the additional views and changes we've made to support features like tags), or the new renderer which does.
As such, having it enabled by default is currently the correct course for current releases to end-users. It could be centralised / externalised though -- but this switch is likely to be removed closer to release, when full feature parity is achieved between the new and old renderers.

Reply:
Please add this as a code comment.  Can you please explain the thinking behind having the debug prefix on the feature toggle variable name, as I find it a little confusing.

Reply:
I've updated this variable to be named "enableNewRoomRendering", and added a comment stating that it's necessary to have this ON for the time being for tags to render.